com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+      "sources":
+      [
+
+        {
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/glibmm/2.58/glibmm-2.58.0.tar.xz",
+          "sha256": "d34189237b99e88228e6f557f7d6e62f767fe356f395a244f5ad0e486254b645"
+        }
+
+      ]
+
+    },
+
+
+
+    {
+
+      "name": "cairomm",
+      "config-opts": [ "--disable-documentation" ],
+      "sources":
+      [
+
+        {
+          "type": "archive",
+          "url": "http://ftp.acc.umu.se/pub/GNOME/sources/cairomm/1.12/cairomm-1.12.0.tar.xz",
+          "sha256": "a54ada8394a86182525c0762e6f50db6b9212a2109280d13ec6a0b29bfd1afe6"
+
+        }
+
+      ]
+
+    },
+
+
+
+    {
+
+      "name": "pangomm",
+      "config-opts": [ "--disable-documentation" ],
+      "sources":
+      [
+
+        {
+
+          "type": "archive",
+          "url": "http://ftp.acc.umu.se/pub/GNOME/sources/pangomm/2.42/pangomm-2.42.0.tar.xz",
+          "sha256": "ca6da067ff93a6445780c0b4b226eb84f484ab104b8391fb744a45cbc7edbf56"
+
+        }
+
+      ]
+
+    },
+
+
+
+    {
+
+      "name": "atkmm",
+      "config-opts": [ "--disable-documentation" ],
+      "sources":
+      [
+        {
+
+          "type": "archive",
+          "url": "http://ftp.gnome.org/pub/GNOME/sources/atkmm/2.28/atkmm-2.28.0.tar.xz",
+          "sha256": "4c4cfc917fd42d3879ce997b463428d6982affa0fb660cafcc0bc2d9afcedd3a"
+
+        }
+
+      ]
+
+    },
+
+
+
+    {
+
+      "name": "gtkmm",
+      "config-opts": [ "--disable-documentation" ],
+      "build-options" : { "cxxflags": "-O2 -g -std=c++17" },
+      "sources": 
+      [
+
+        {
+
+          "type": "archive",
+          "url": "http://ftp.acc.umu.se/pub/GNOME/sources/gtkmm/3.24/gtkmm-3.24.0.tar.xz",
+          "sha256": "cf5fc92805e581c8303e08d54519457ba07f15b766e9b1edde4862993ac1aa43"
+
+        }
+
+      ]
+
+    }, 
+
+
+
+    {
+
+      "name": "clastfm",
+
+      "build-options" : { "cxxflags": "-O2 -g -std=c++17" },
+
+      "sources":
+      [
+
+        {
+
+          "type": "git",
+          "url": "git://liblastfm.git.sourceforge.net/gitroot/liblastfm/liblastfm"
+
+        }
+
+      ]
+
+    },
+
+
+
+    {
+
+      "name": "libconfig",
+      "sources":
+      [
+
+        {
+
+          "type": "archive",
+          "url": "https://hyperrealm.github.io/libconfig/dist/libconfig-1.7.2.tar.gz",
+          "sha256": "7c3c7a9c73ff3302084386e96f903eb62ce06953bb1666235fac74363a16fad9"
+
+        }
+
+      ]
+
+    },
+
+
+
+    {
+
+      "name": "omp",
+      "buildsystem": "simple",
+      "build-options" : { "cxxflags": "-g -std=c++17 -lstdc++fs -DFLATPAK=$(FLATPAK)", "prefix": "/app" },
+      "build-commands":
+      [
+
+        "make -e FLATPAK=1 -j12",
+        "make DESTDIR=/app install"
+
+      ],
+      "sources":
+      [
+
+        {
+
+          "type": "git",
+          "url": "https://github.com/TheWiseNoob/OMP.git"
+
+        }
+
+      ]
+
+    }
+
+  ]
+
+}
```

Comment:
Please use sane formatting. If you don't like [YAML](https://blogs.gnome.org/alexl/2018/05/04/yamling-the-flathub), at least get rid of empty lines, and put list elements on separate lines.